boilerplate/package.json

```diff
@@ -97,6 +98,9 @@
     "ts-node": "^10.9.2",
     "typescript": "~5.3.3"
   },
+  "resolutions": {
+    "eslint-plugin-react": "7.35.2"
```

Comment:
What was the reason for adding this?

If we're going to add a yarn resolution we also need to add the proper keys for other package managers, such as [overrides for npm](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides)

Reply:
This I ended up adding in the expo-router PR. But this is the commit: https://github.com/infinitered/ignite/commit/b1cb6c3dd47a236baf2aa2ec375ce7bf19746a3a

Today there were new test failures because they published a new version of this package that has a bug in a rule, but the fix wasn't published yet: https://github.com/jsx-eslint/eslint-plugin-react/pull/3821.

I was trying to pin it this way since it's not a direct dependency, but didn't realize that didn't work for all package managers. My bad.

@fpena we'll want to remove this specific change from you PR; you added this in a different spot in package.json and now if you look at the file there's two separate `resolutions` fields.

Reply:
Actually it looks like we can remove both `resolutions` sections, they published a new version that fixes the issue: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/CHANGELOG.md#7361---20240912

Reply:
@lindboe @frankcalise  Yes, removed them from this PR.

Reply:
Cool thanks!